vpr/src/server/task.h

```diff
@@ -19,36 +19,162 @@ namespace server {
  */
 class Task {
 public:
+    /**
+     * @brief Constructs a new Task object.
+     * 
+     * @param job_id The ID of the job associated with the task.
+     * @param cmd The command associated with the task.
```

Comment:
Suggest listing the valid cmd values and what each means, or inserting a ref to that info in index.rst.

Reply:
Task class now uses comm::CMD enum as cmd instead of int, ref to enum structure is added in few places of Task.h.
solved
e3cff2771f698995601293c6114bb96a5431dcc0
3b2eb6a2e435ada504e7c9e276fae1e10ea57449